react-props/react-props-and-expressions-quiz-notes.md

```diff
@@ -6,12 +6,20 @@ Answer the following questions in the provided markdown file before turning in t
 
 - What are props in React?
 
+It is an object which allows for the passing of arguments to react components
+
 - How do you declare and access props in a component?
 
+You declare the props using the type keyword and access them using object destructuring in the parameter of the component definition
```

Comment:
The type keyword is how we annotate or type our props, but how do we actually declare them?